src/main.rs

```diff
@@ -1,9 +1,13 @@
+use crate::usage::usage;
 use std::{collections::HashMap, error::Error, io::Write, num::ParseIntError, ops::AddAssign};
 
+mod usage;
+
 fn main() -> Result<(), Box<dyn Error>> {
     let mut opts = getopts::Options::new();
     let opts = opts
-        .optflag("q", "quiet", "Hides the output of commit messages")
+        .optflag("q", "quiet", "Hides the output of commit messages.")
+        .optflag("h", "help", "Show this help and exit.")
```

Comment:
Yes, this also happened to me.  I expected the CLI to have a help option and was confused that it was considered a failure.  So, I added this feature since I considered it a meaningful change.

Reply:
By the way, how about adding an option overview to the README?

Reply:
Sure, why not 😄